File: How-Many-Numbers-Are-Smaller-Than-the-Current-Number.py
class Solution(object):
    def smallerNumbersThanCurrent(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        count = 0
        output = []
        for i in range (0, len(nums)):
            for j in range (0, len(nums)):
                if j != i and nums[j] < nums[i]:
                    count += 1
            output.append(count)
            count = 0
        return output

File: how-many-numbers-are-smaller-than-the-current-number.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int a = nums.length;
        int count = 0;
        int[] counts = new int[a];
        int k = 0;
        for (int i = 0; i < a; i++) {
            for (int j = 0; j < a; j++) {
                if (j != i && (nums[j]<nums[i])) {
                    count++;
                }
            }
            counts[k++] = count;
            count = 0;
        }
        return counts;
    }
}
